templates/components/products/carousel.html

```diff
@@ -3,31 +3,8 @@
         "dots": true,
         "infinite": false,
         "mobileFirst": true,
-        "slidesToShow": 2,
-        "slidesToScroll": 2,
-        "responsive": [
-            {
-                "breakpoint": 1260,
-                "settings": {
-                    "slidesToScroll": 3,
-                    "slidesToShow": {{ columns }}
-                }
-            },
-            {
-                "breakpoint": 800,
-                "settings": {
-                    "slidesToScroll": 3,
-                    "slidesToShow": 5
-                }
-            },
-            {
-                "breakpoint": 550,
-                "settings": {
-                    "slidesToScroll": 3,
-                    "slidesToShow": 3
-                }
-            }
-        ]
+        "slidesToShow": {{ columns }},
```

Comment:
Maybe we should consider hard-coding this to 2 or 3 to give better backward compatibility? @bigcommerce/storefront-team @bigcommerce/merc-team

Reply:
That was my thought as well, just go with 3 to keep thing as consistent as possible (though I don't really see why full page scrolls would be undesirable)